.github/workflows/e2e.yaml

```diff
@@ -110,3 +110,6 @@ jobs:
       - name: Run e2e tests
         run: |
           sudo GINKGO_FOCUS="${{ matrix.ginkgo_focus }}" SKIP_RESOURCE_CLEANUP=true make test-e2e
+      - name: Setup tmate session
```

Comment:
will keep this in for some more runs to be able to debug things in case something breaks. Will be removed before merge.